spec/integration/util/windows/adsi_spec.rb

```diff
@@ -33,7 +33,7 @@
 
   describe '.[]' do
     it 'should return string attributes as UTF-8' do
-      administrator = Puppet::Util::Windows::ADSI::User.new('Administrator')
+      administrator = Puppet::Util::Windows::ADSI::User.new('Guest')
```

Comment:
```suggestion
      guest = Puppet::Util::Windows::ADSI::User.new('Guest')
```

Reply:
added the suggested change in https://github.com/puppetlabs/puppet/pull/8351/files
I was thinking to have that merged first so the tests work on all environments. I'll update this PR after that gets merged